mediapipe/tasks/c/components/containers/embedding_result_converter.h

```diff
@@ -29,6 +29,10 @@ void CppConvertToEmbeddingResult(
     const mediapipe::tasks::components::containers::EmbeddingResult& in,
     EmbeddingResult* out);
 
+void ConvertToCppEmbedding(
```

Comment:
This is a C++ function so it should start with Cpp, even though `CppConvertToCppEmbedding` is not going to win us any awards :)